stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -430,7 +430,7 @@ public class MessageListView : ConstraintLayout, IMessageListView {
                     firstVisiblePosition = currentFirstVisible
 
                     val realLastVisibleMessage =
-                        min(max(currentLastVisible, getLastSeenMessagePosition()), currentList.size)
+                        min(max(currentLastVisible, getLastSeenMessagePosition()), currentList.size - 1)
```

Comment:
Would this be the same logic?

```suggestion
                       max(currentLastVisible, getLastSeenMessagePosition()).coerceAtMost(currentList.lastIndex)
```

Reply:
neat!